### ISSUE TITLE
[FEATURE] Ability to "re-send" invite emails to members of a round and group

### ISSUE THREAD

**Issue by francesca-sp** [cobudget/cobudget#534]
**Describe the feature**
Right now, it is only possible to send out an invitation to people to join a Cobudget round or group once. This is problematic, because it is common that such an invite email might get lost, it ends up in a spam folder or is deleted by mistake. Therefore it is an important feature for the group admin to be able to resend an invite. 
![image](https://user-images.githubusercontent.com/3641828/151818580-15048cec-a657-4fc5-bd50-7535d8bb9dd3.png)

It could look similar to in legacy cobudget. When you click on the three dots on the right of a member, you add an additional action next to "make admin" called "invite again" or "resend invite". 

![image](https://user-images.githubusercontent.com/3641828/151818305-cc8c5573-7ac4-42ab-b6bb-0b000b7d2f4f.png)

**Comment by aerugo** [comment]
This is a duplicate of #443 - I will move it to Member management milestone and replace that issue with this one.

**Comment by gustavlrsn** [comment]
The workaround of simply inviting them again is working for Rounds but not Groups, which is a bug tracked here #535

**Comment by alinauroz** [comment]
What is the correct way to detect a pending invite?

If an invite is sent to a user who has not joined Cobudget, it means the invite is pending. Right?

**Comment by aerugo** [comment]
An invite should be pending until the user clicks the link, and this action should change the users membership in the round from pending.

Users may still have joined Cobudget and be pending on a specific round, because the invite is for them to join that round, not Cobudget as a whole.

**Comment by aerugo** [comment]
Note that this is different than “verified” which a user only needs to do once to verify their email once they register on cobudget. So there are two states: Verified (global) and pending (local).

**Comment by alinauroz** [comment]
Right now, when an invitation is sent to a user who is already registered on cobudget, no approval is required. 

For example ali@ex.com is a member of cobudegt. When an invitation is sent to ali@ex.com, this user is immediately added to the collection.

**Comment by alinauroz** [comment]
Approvals are only pending when a user is has not joined cobudget.

**Comment by aerugo** [comment]
Yes, this is true. A better system for handling pending users is on the TODO. For now, I would add a simple “pending” to the member table and flip that with a link in the invite email.

**Comment by aerugo** [comment]
I have discussed with @alinauroz and this issue is actually a little larger in scope than we thought.
We need to add an invitations table to store unique invite links to the collection. He will go ahead and do that too.

**Comment by alinauroz** [comment]
Final Solution:

1. Budget can be allocated to a member even if the member's invitation status is still pending.
2. Extra column will be added CollectionMember table to check the invitation status.
3. User will not be considered a member if he has not accepted the invitation. If the invitation is pending user will not be able to fund or comment.
4. There will be a "Accept Invitation" button on collection page if the user has not accepted the invitation.

**Comment by aerugo** [comment]
> User will not be considered a member if he has not accepted the invitation.

Remember however that a user can also freely join a collection without an invitation if the settings allow that.

> If the invitation is pending user will not be able to fund or comment.

User should also not be able to create buckets until not pending.

So @alinauroz - we are changing the idea to users not joining directly through clicking the link, but instead they will get linked to the member page and need to click the accept invitation button by their name? Or would following the link still automatically accept for them?
